package.json

```
@@ -118,8 +118,10 @@
     "update-snapshot": "esmocha --no-insight --no-parallel --update-snapshot --",
     "update-snapshots": "esmocha test generators cli .blueprint lib/jdl lib --forbid-only --update-snapshot --enable-source-maps"
   },
+  "overrides": {
+    "@eslint/core": "1.0.0"
```

Avoid having 2 versions.
```suggestion
    "@eslint/core": "$@eslint/core"
```